lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/qm/XwikiBridgeInterfaceRestResource.java

```diff
@@ -22,51 +22,61 @@
 import org.xwiki.component.annotation.Component;
 import org.xwiki.rest.XWikiRestComponent;
 
-import eu.learnpad.exception.LpRestException;
-import eu.learnpad.exception.impl.LpRestExceptionImpl;
+import eu.learnpad.exception.impl.LpRestExceptionXWikiImpl;
 import eu.learnpad.qm.BridgeInterface;
 import eu.learnpad.core.rest.RestResource;
 
+/*
+ * The methods inherited form the BridgeInterface in this
+ * class should be implemented as a REST invocation
+ * toward the BridgeInterface binded at the provided URL
+ */
 @Component
```

Comment:
As I told in the email talking about Components, these components will be inject in the code by the component's tools.  It means that only the empty constructor will be called.  Therefore, any constructor with parameters will never be called.